frontend/src/metabase/actions/components/ActionViz/ActionButton.styled.tsx

```diff
@@ -2,18 +2,23 @@ import styled from "@emotion/styled";
 import { color } from "metabase/lib/colors";
 
 import Button from "metabase/core/components/Button";
+import { breakpointMinLarge } from "metabase/styled-components/theme";
 
 export const StyledButton = styled(Button)<{
   isFullHeight?: boolean;
   focus?: boolean;
 }>`
   height: ${({ isFullHeight }) => (isFullHeight ? "100%" : "auto")};
+
   ${({ focus }) =>
     focus
       ? `
     border: 2px solid ${color("focus")};
   `
       : ""}
+  ${breakpointMinLarge} {
```

Comment:
What about lower resolutions? Shouldn't we always remove the padding?

Reply:
I'm worried using breakpoints will only increase more surface area where bugs could occur. With the responsive nature of our layout we could
1. toggle the side nav
2. toggle subscription sidebar
3. toggle more info sidebar (they don't have the same width atm)

That means the dashboard container size isn't bound by the viewport and vary depending on what elements are visible on the UI.

I think we should try to solve this problem without breakpoint if that makes sense.

Reply:
Yeah, this was a weird one - for some reason it's just this breakpoint that was causing problems. I tested just having `padding: 0` and it seemed to work. I can just put that, just wanted to be surgical about it 😄

Reply:
Ok, just reran the tests with the `padding: 0` and things are still working as expected!